.appveyor.yml

```diff
@@ -17,7 +17,7 @@ environment:
   SBT_VERSION: 1.0.2
   STACK_VERSION: 2.1.3.20190715
   VIRTUAL_ENV_VERSION: 15.1.0
-  YARN_VERSION: 1.17.3
+  YARN_VERSION: 1.21.1
```

Comment:
Just wondering, is there a concrete reason for the upgrade? Like older Yarn versions not being available anymore in some distros?

Reply:
No, it's just to keep our dependencies up to date.